index.md

```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
 
 ```
 wget https://raw.githubusercontent.com/kiali/kiali/master/hack/cluster-openshift.sh
-sh cluster-openshift.sh --istio-version 0.7.1 --kiali-enabled true up
+sh cluster-openshift.sh --istio-version 0.8 --kiali-enabled true up
 ```
 
 [NOTE]
-Notice the above command line forces the version of Istio to 0.7.1. Kiali will support Istio 0.8.0 in the very near future, but for now, we recommend using --istio-version 0.7.1 if you just want to see what Kiali is all about.
+Notice the above command line forces the version of Istio to 0.8 Kiali will support Istio 0.8.0 in the very near future, but for now, we recommend using --istio-version 0.8 if you just want to see what Kiali is all about.
```

Comment:
This comment now doesn't make sense. I think we should either remove this note (lines 114-5) completely or change it to say something like "we are recommending 0.8.0 in the above command, but keep in mind Kiali is currently in active development to get it to support 0.8.0 fully."